src/pip/_internal/utils/parallel.py

```diff
@@ -68,6 +75,17 @@ def _map_fallback(func, iterable, chunksize=1):
     return map(func, iterable)
 
 
+def _map_ordered_fallback(func, iterable, chunksize=1):
+    # type: (Callable[[S], T], Iterable[S], int) -> List[T]
+    """Make a list applying func to each element in iterable.
+
+    This function is the sequential fallback either on Python 2
```

Comment:
Pip no longer supports Python 2.

Reply:
Hmm copy pasted from the unordered doc-string :) I'll fix on both